distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -531,6 +531,57 @@ type Decimal
     negate : Decimal
     negate self = Decimal.Value self.big_decimal.negate
 
+    ## GROUP CONVERSIONS
+       Convert this to an `Integer`.
+
+       Conversion to `Integer` uses truncation: the fractional portion is
+       dropped.
+
+       > Example
+         Convert `Decimal` 2345.6 to an `Integer`.
+             d = Decimal.new "2345.6"
+             d.to_integer
+             # => 2345
+    to_integer : Integer
+    to_integer self = self.big_decimal.toBigInteger
+
+    ## GROUP CONVERSIONS
+       Convert this to a `Float`.
+
+       ? Precision Loss
+
+         While any `Float` can be precisely represented by a `Decimal`, not
+         every `Decimal` can be accurange represented by a `Float`, so this
+         conversion can lose precision. For this reason, converting a
+         `Decimal` to a `Float` always attaches a `Loss_Of_Numeric_Precision`
+         warning to the result.
+
+       ? Representable Values
+
+         If the `Decimal` is outside the representable range of `Float` (which
+         is `(-Double.MAX_VALUE, Double.MAX_VALUE)`, inclusive), `+/-Inf` is
+         returned, and an `Out_Of_Range` warning is attached.
+
+       ! Error Conditions
+
+         - If the `Decimal` is outside the representable range of `Float` (which
+           is `(-Double.MAX_VALUE, Double.MAX_VALUE)`, inclusive), `+/-Inf` is
+           returned, and an `Out_Of_Range` warning is attached.
+         - If the conversion results in a loss of precision, a
+           `Loss_Of_Numeric_Precision` warning is attached.
+
+       > Example
+         Convert `Decimal` 23.45 to a `Float`.
+
+             d = Decimal.new "23.45"
+             d.to_float
+             # => 23.45
+    to_float : Integer
+    to_float self =
```

Comment:
Why there is `dec.to_float` when there also is a conversion and one can write `dec.to Float`?

Reply:
I think the idea was that currently there are no good completions for `.to` conversions - the IDE will not easily tell you which conversions are available and which ones are not. Having the explicit ones allows it to show up in the Component Browser.